index.html

```diff
@@ -7,21 +7,26 @@
   <script class="remove">
     // All config options at https://respec.org/docs/
     var respecConfig = {
-      specStatus: 'editor-draft-finding',
+      specStatus: 'DNOTE',
       group: 'tag',
+      publishDate: '2022-04-27',
       format: 'markdown',
       editors: [{
         name: 'Robin Berjon',
         company: 'The New York Times',
         companyURL: 'https://nytimes.com/',
         url: 'https://berjon.com/',
+        w3cid: 34327,
       }, {
         name: 'Jeffrey Yasskin',
         company: 'Google',
         companyURL: 'https://google.com/',
       }],
       github: 'w3ctag/privacy-principles',
-      shortName: 'privacy-principles',
+      shortName: 'privacy',
```

Comment:
```suggestion
      shortName: 'privacy-principles',
```
I'm not certain we need the longer shortname, but I think these are global identifiers across all of W3C, and I don't think this is the only thing that covers "privacy".